src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -47,48 +50,106 @@ const HousingInfo = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={handleSubmit}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '8px',
+        justifyContent: 'space-between'
+        // maxWidth: '720px',
+      }}
+    >
       <div
         style={{
-          padding: '8px',
           display: 'flex',
-          flexDirection: 'column',
           gap: '8px',
-          maxWidth: '360px'
+          minWidth: '846px',
+          padding: '8px'
+        }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+            padding: '8px',
+            width: '40%'
+          }}
+        >
+          <TextField
+            id="street-input"
+            name="lastPermanentStreet"
+            label="Street:"
+            onChange={handleChange}
+            value={formData.lastPermanentStreet}
```

Comment:
For `formData`, the values from keys could potentially be `null`. In case of nulls, we should make the text fields fall back to an empty string with `??`.

Reply:
@leekahung — I have further questions on this. how could the `formData` values become null? We initialize them in 18 through 28 with an empty string, `'',` which is falsy, but not `null`. If a user does not fill anything in for that field, and they submit, that field will be sent to Interrupt and then the Solid data store as an empty string, but not `null`. In what way can the value become null?

Reply:
That is because the fetched value from Solid could be `null` which replaces the initial state from `useState` with `null` after running `useEffect`. See how `useEffect` takes in `data` as one of its dependencies in its dependency array.

Reply:
In case you're uncertain, feel free to check the console or use console.log, which would be useful to confirm `formData`. I recommend using console logs for debugging.

![Screenshot 2024-05-22 at 6 51 28 PM (3)](https://github.com/codeforpdx/PASS/assets/14917816/e7299bca-1bba-436b-9a64-f3f933277983)

![Screenshot 2024-05-22 at 6 51 48 PM (3)](https://github.com/codeforpdx/PASS/assets/14917816/0de9c20b-13d9-4d8e-8d79-378301c813ef)

Reply:
> That is because the fetched value from Solid could be `null` which replaces the initial state from `useState` with `null` after running `useEffect`. See how `useEffect` takes in `data` as one of its dependencies in its dependency array.

@leekahung — okay, so where should i put this nullish coalescing operator `??`, in `HousingInfo.jsx` on lines 22-30 where we initialize each of these fields into `useState`, or on each of the `<TextField>`'s for their `value` prop?

Reply:
Line 85 is one such spot where `value={formData.lastPermanentStreet ?? ''}`. You can initialize the rest of the TextFields where there are potential nulls coming from Solid like street, city, state, zip code.

Reply:
Okay, now pushed to this branch, `513-Enhancement-Civic-Profile-Housing-Information-Form`, this change with the `??` operator to prevent null. Fixed.